bigbone/src/main/kotlin/social/bigbone/api/entity/Notification.kt

```diff
@@ -48,10 +48,18 @@ data class Notification(
     /**
      * Specifies the notification type.
      */
-    enum class Type(val value: String) {
-        Mention("mention"),
-        Reblog("reblog"),
-        Favourite("favourite"),
-        Follow("follow")
+    @Serializable
+    enum class NotificationType {
```

Comment:
This type is missing `status`, `follow_request`, `poll`, `update`, `admin.sign_up`, and `admin.report` but we should probably only add them in a follow-up PR where we check all notification types again.

Reply:
Created #313